### PR TITLE
add create function to Author and OntologyAnnotation

### PR DESCRIPTION
**Pull request by caroott**
This PR adds the "create" function to the types Author and OntologyAnnotation for easier type initialization

### CODE REVIEW

File: src/AVPRIndex/Domain.fs [hunk 1]
```diff
@@ -39,6 +39,21 @@ module Domain =
                 )
             | _ -> false
 
+        static member create (
+            fullName: string,
+            email: string,
+            affiliation: string,
```
Comment (kMutagene) [inline src/AVPRIndex/Domain.fs:45]: email, affiliation, and affiliation link should be options

File: src/AVPRIndex/Domain.fs [hunk 2]
```diff
@@ -66,6 +81,19 @@ module Domain =
                 )
             | _ -> false
 
+        static member create (
+            name: string,
+            termSourceRef: string,
```
Comment (kMutagene) [inline src/AVPRIndex/Domain.fs:86]: ref and tan should be optional

**Review by kMutagene — CHANGES_REQUESTED** [review]

File: src/AVPRIndex/Domain.fs (hunk 1 above)
Comment (caroott) [reply src/AVPRIndex/Domain.fs]: done

File: src/AVPRIndex/Domain.fs (hunk 2 above)
Comment (caroott) [reply src/AVPRIndex/Domain.fs]: done

**Review by kMutagene — APPROVED** [review]

**Pull request merged**